packages/typography/src/Overline/Overline.tsx

```diff
@@ -1,32 +1,41 @@
 import React from 'react';
 
-import Box, { ExtendableBox } from '@leafygreen-ui/box';
 import { cx } from '@leafygreen-ui/emotion';
 import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  Polymorphic,
+  PolymorphicAs,
+  usePolymorphic,
+} from '@leafygreen-ui/polymorphic';
 
 import { baseTypographyStyles, defaultTextColor } from '../styles';
 
 import { overlineStyles } from './Overline.styles';
 import { OverlineProps } from './Overline.types';
 
-export const Overline: ExtendableBox<OverlineProps, 'div'> = ({
-  darkMode: darkModeProp,
-  className,
-  ...rest
-}: OverlineProps) => {
-  const { theme } = useDarkMode(darkModeProp);
-  return (
-    <Box
-      className={cx(
-        baseTypographyStyles,
-        overlineStyles,
-        defaultTextColor[theme],
-        className,
-      )}
-      {...rest}
-    />
-  );
-};
+export const Overline = Polymorphic<OverlineProps>(
```

Comment:
Should `Disclaimer` also extend Polymorphic?

Reply:
We had a ticket to convert Label which is why I did.. happy to add Disclaimer if you feel it's necessary?

Reply:
Don't think it's necessary, no. Just wanted to make sure 👍